EndOfRunMonitor/ISISendOfRunMonitor.py

```diff
@@ -18,7 +18,7 @@
 DATA_LOC = r"\data\cycle_%s\\"
 SUMMARY_LOC = r"\logs\journal\SUMMARY.txt"
 LAST_RUN_LOC = r"\logs\lastrun.txt"
-LOG_FILE = r"monitor_log.txt"
+LOG_FILE = r"C:\monitor_log.txt"
```

Comment:
Do we want logging to go to `C:\`? Would it be better to have it in the project somewhere e.g. `EndOfRunMonitor/monitor_log.txt`. This could be done with something like: `os.path.join(os.path.dirname(os.path.abspath(__file__)))`